### DISCUSSION TITLE
Comments module - enable breaking lines

### DISCUSSION THREAD

**Discussion by s2b-git** [YetiForceCompany/YetiForceCRM#14948] [Feature improvement]
Comments (especially in tickets) are often used as a way of communication. In yetiforce here is little imperfection as we cannot break line while composing comment. html tag br also not working. It would be helpful if comments can be extended to this point.

Best

**Comment by rskrzypczak** [comment]
Simple ENTER not working for you?
What version are you using?
In the current stable version (6.2) I don't see a problem here.
![image](https://user-images.githubusercontent.com/10221999/125229525-421d6200-e2d7-11eb-98de-e6379e821ae1.png)

**Reply by s2b-git** [reply]
The problem is in variables (emails templates). Text pulled out from the field got stripped brutally from all beauty :-)

best
